rules/linux/process_creation/proc_creation_lnx_clear_logs.yml

```diff
@@ -22,6 +22,10 @@ detection:
         CommandLine|contains:
             - '/var/log'
             - '/var/spool/mail'
+            - '/var/lib/docker/containers'
+            - '/var/lib/mysql'
+            - '/var/lib/pgsql'
```

Comment:
I don't understand the reasoning behind adding these to this rule? These data paths and not log paths